src/bioregistry/data/bioregistry.json

```diff
@@ -104,11 +104,32 @@
       "name": "Charles Tapley Hoyt",
       "orcid": "0000-0003-4423-4370"
     },
+    "contributor_extras": [
+      {
+        "email": "laylamichan@ciencias.unam.mx",
+        "github": "lmichan",
+        "name": "Layla Michán Aguirre",
+        "orcid": "0000-0002-5798-662X"
+      },
+      {
+        "email": "mineromero2901@ciencias.unam.mx",
+        "github": "MinRo60",
+        "name": "Minerva María Romero Pérez"
```

Comment:
ORCIDs are required for all attributions - note this is also encoded in the unit tests to ensure data integrity